.circleci/config/jobs/fmt.yml

```diff
@@ -0,0 +1,13 @@
+executor: go-machine
+steps:
+- checkout
+- install-go
+- run:
+    name: "Install tools"
+    command: make tools
+- run:
+    name: "Format files"
+    command: make fmt
```

Comment:
This should be covered by [make-gen-deltas](https://github.com/hashicorp/boundary/blob/1342bd018317a44fd9f97dd93c31523941306c29/.circleci/config/jobs/make-gen-deltas.yml) since `make fmt` is call via `make gen`

Reply:
Good catch!

Reply:
Removed